AllJoyn/Platform/BridgeHost/App.xaml.cs

```diff
@@ -0,0 +1,359 @@
+﻿//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+
+using System;
+using Windows.ApplicationModel.Activation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+using Windows.ApplicationModel.Background;
+using Windows.ApplicationModel.AppService;
+using Windows.Foundation.Collections;
+using System.Linq;
+using BridgeRT;
+using Windows.Storage;
+using Windows.ApplicationModel;
+using System.Text;
+using Windows.ApplicationModel.DataTransfer;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Runtime.Serialization.Json;
+using System.Threading.Tasks;
+using System.Diagnostics;
+
+// The Blank Application template is documented at http://go.microsoft.com/fwlink/?LinkId=402347&clcid=0x409
+
+namespace OpenT2T.Bridge
+{
+    [DataContract]
+    internal class DeviceProps
```

Comment:
I am not sure. This basically should map to DeviceInfo in BridgeRT. Currently other properties are not send to app service (nor it expects). future enhancements (added to readme.md)

public sealed class DeviceInfo : __IDeviceInfoPublicNonVirtuals
    {
        [Overload("CreateInstance1")]
        public DeviceInfo();

        public string Description { get; set; }
        public string FirmwareVersion { get; set; }
        public string Model { get; set; }
        public string Name { get; set; }
        public string Props { get; set; }
        public string SerialNumber { get; set; }
        public string Vendor { get; set; }
        public string Version { get; set; }
    }

---
In reply to: [75187292](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75187292) [](ancestors = 75187292)